### PR TITLE
fix #2254 can't add emojis with enter in reflections

### LINKED ISSUE #2254: Special submit handling for task and reflection card emoji menu
## Issue - Bug

When I type ":" to begin entering an emoji on a Reflection card, type some letters and press enter, the portal closes without inputting the emoji.

See also: #2179 

- **OS:** macOS
- **Browser:** Chrome
- **Estimated effort:** 10.5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### Acceptance Criteria (optional)
On the Task _and_ Reflection cards:
   - Press enter to submit an emoji
   - Press tab to submit an emoji
   - Press shift-enter (sure, why not?) to submit an emoji
   - Can click on emoji with mouse cursor

### PR DESCRIPTION
**Pull request by mattkrick**
TEST

- write a reflections, hit `:` select an emoji with the arrow keys & press enter. the emoji is added. press enter to submit the card. it works

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh ready for review

File: src/universal/components/ReflectionEditorWrapper.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ class ReflectionEditorWrapper extends PureComponent<Props> {
   }
 
   handleReturn = (e) => {
-    const {handleReturn} = this.props
-    if (handleReturn) {
+    const {handleReturn, renderModal} = this.props
```
Comment (jordanh) [inline src/universal/components/ReflectionEditorWrapper.js]: Ha. Sometimes it's the simple stuff

**Comment by jordanh** [comment]
# Test Cycle 1

On the Task cards:

- [x] Press enter, tab, or shift-enter to submit an emoji
- [ ] ❌ Can click on emoji with mouse cursor

On the Reflection cards:

- [x] Press enter, tab, or shift-enter to submit an emoji
- [ ] ❌ Can click on emoji with mouse cursor

**Comment by jordanh** [comment]
@mattkrick., getting better! Still one small thing to fix... can we catch that click event somewhat easily?

**Pull request merged**